src/style.css

```diff
@@ -1 +1,24 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
```

Comment:
Consider using a class for the `stars__star` elements that specifies the default star state (e.g., `stars__star--inactive`), and then override the `background-image` for the active state in the `.stars--1 .stars__star:nth-child(-n + 1)` and similar selectors. This will make the CSS more readable and maintainable by clearly distinguishing between the default and active states.